Sources/UI/SwiftUI/Views/PhotoPreview/PhotoPreviewConfiguration.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PhotoPreviewConfiguration.swift
+//  PovioKit
+//
+//  Created by Ndriqim Nagavci on 09/09/2024.
+//  Copyright © 2024 Povio Inc. All rights reserved.
+//
+
+import SwiftUI
+
+public struct PhotoPreviewConfiguration {
+  let backgroundColor: Color
+  let showDismissButton: Bool
+  let velocityThreshold: CGSize
+  let offsetThreshold: CGFloat
+}
```

Comment:
A public `init` is missing. I would also make the properties public and mutable.